app/src/hooks/usePpg.ts

```diff
@@ -85,36 +87,19 @@ const usePgp = (): Hooks => {
     try {
       if (!publicKey || !email) return;
       // Upload public key to openpgp server
-      const uploadResponse = await axios.post<UploadPGPKeyResponse>(
-        "https://keys.openpgp.org/vks/v1/upload",
-        JSON.stringify({
-          keytext: publicKey
-        }),
-        {
-          headers: {
-            "Content-Type": "application/json"
-          }
-        }
-      );
-
+      const uploadResponse = await api.publishPGPKey(publicKey);
       // Verify key,send email
-      const verifyResponse = await axios.post(
-        "https://keys.openpgp.org/vks/v1/request-verify",
-        JSON.stringify({
-          token: uploadResponse.data.token,
-          addresses: [email]
-        }),
-        {
-          headers: {
-            "Content-Type": "application/json"
-          }
-        }
-      );
+      const verifyResponse = await api.verifyPGPKey({
+        token: uploadResponse.token,
+        addresses: [email]
+      });
 
-      if (verifyResponse.status === 200) {
+      if (verifyResponse) {
+        await updateClaim("EmailCredential", { email: email }, [], false);
```

Comment:
`EmailCredential`, `NameCredential`, etc should be constants instead of re-typing string literals into the source code.  This enables the compiler to pick up typos.